openeo_processes_dask/process_implementations/arrays.py

```diff
@@ -81,3 +98,196 @@ def count(data: RasterCube, condition: Callable, **kwargs):
     else:
         data = data.sum()
     return data
+
+
+def array_create(data: Optional[ArrayLike] = None, repeat: Optional[int] = 1):
+    if data is None:
+        data = np.array([])
+    return np.tile(data, reps=repeat)
+
+
+def array_modify(
+    data: ArrayLike,
+    values: ArrayLike,
+    index: int,
+    length: Optional[int] = 1,
+):
+    if index > len(data):
+        raise ArrayElementNotAvailable(
+            "The array can't be modified as the given index is larger than the number of elements in the array."
+        )
+    first = data[:index]
+    modified = np.append(first, values)
+    if index + length < len(data):
+        modified = np.append(modified, data[index + length :])
+    return modified
+
+
+def array_concat(array1: ArrayLike, array2: ArrayLike):
+    concat = np.append(array1, array2)
+    return concat
+
+
+def array_contains(data: ArrayLike, value: Any):
+    if type(value) not in [int, float, str] or pd.isnull(value):
+        return False
+    if len(np.shape(data)) != 1:
+        return False
+    else:
+        return np.isin(data, value).any()
+
+
+def array_apply(
+    data: ArrayLike,
+    process: Callable,
+    context: Optional[dict] = None,
+):
+    context = context if context is not None else {}
+    if not hasattr(data, "__array_interface__"):
+        data = np.array(data)
+    return process(data, **context)
```

Comment:
Oh, sorry, this is an implementation based on the outdated version of handling the `from_parameter`, that is only there, because I started updating the processes before the latest `from_parameter` updates were added. (Which is also why there are no tests for the process.) This is actually the same for the `array_filter` and the `count` process for now.

Reply:
Just created https://eodc.atlassian.net/browse/EODCAPI-233 to address this separately!